The job **`x86_64-mingw`** failed! Check out the build log: [(web)](https://github.com/rust-lang-ci/rust/actions/runs/11414877171/job/31764330129) [(plain)](https://github.com/rust-lang-ci/rust/commit/eaf7f2c8512a3b0f4e0c26a73ce055c381759b8e/checks/31764330129/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
test [run-make] tests\run-make\zero-extend-abi-param-passing ... ok
test [run-make] tests\run-make\test-benches ... ok
test [run-make] tests\run-make\share-generics-export-again ... ok
test [run-make] tests\run-make\rustc-crates-on-stable ... ok
Terminate batch job (Y/N)? 
##[error]The operation was canceled.
Post job cleanup.
[command]"C:\Program Files\Git\bin\git.exe" version
git version 2.46.2.windows.1
```

</details>

--- a/library/alloc/Cargo.toml
+++ b/library/alloc/Cargo.toml
@@ -10,7 +10,7 @@ edition = "2021"
 
 [dependencies]
 core = { path = "../core" }
-compiler_builtins = { version = "0.1.133", features = ['rustc-dep-of-std'] }
+compiler_builtins = { version = "0.1.134", features = ['rustc-dep-of-std'] }
 
 [dev-dependencies]
 rand = { version = "0.8.5", default-features = false, features = ["alloc"] }

--- a/library/std/Cargo.toml
+++ b/library/std/Cargo.toml
@@ -17,7 +17,7 @@ cfg-if = { version = "1.0", features = ['rustc-dep-of-std'] }
 panic_unwind = { path = "../panic_unwind", optional = true }
 panic_abort = { path = "../panic_abort" }
 core = { path = "../core", public = true }
-compiler_builtins = { version = "0.1.133" }
+compiler_builtins = { version = "0.1.134" }
 unwind = { path = "../unwind" }
 hashbrown = { version = "0.15", default-features = false, features = [
     'rustc-dep-of-std',

